Chrysalis-Papilio/Soen-490-Papilio
# User login

**Error handling for user authentication**

What to test:
- Verify that error messages related to firebase or network errors are displayed in a snackbar in both sign and login activities

How to do it:
- Try to authenticate while having no internet connection
- Try to authenticate while having a disabled account on firebase

Testing:
- I will work on testing later on in the sprint
